react/Minicart.tsx

```diff
@@ -20,6 +20,7 @@ interface MinicartProps {
   maxDrawerWidth: number | string
   drawerSlideDirection: SlideDirectionType
   quantityDisplay: MinicartIconButtonType
+  showTotalItemsQty: MinicartTotalItemsType
```

Comment:
Now sure on the name of this prop 🤔 
How about `itemCount`? This makes it more intuitive and makes more sense for an `enum`.

Reply:
showTotalItemsQty is more intuitive for me than "itemCount" which I think is too generic. But you have your own rules. If you want to change it that way I can do it.

Reply:
What do you think about `itemCountMode`?